src/UserSpaceInstrumentation/InstrumentProcess.cpp

```diff
@@ -66,6 +67,17 @@ bool ProcessWithPidExists(pid_t pid) {
   return result.value();
 }
 
+// We need to initialize some thread local memory when entering the payload functions. This leads to
+// a situation where instrumenting the functions below would lead to a recursive call into the
+// instrumentation. We just silently skip these and leave instrumenting them to the kernel/uprobe
+// fallback.
+bool IsBlacklisted(std::string_view function_name) {
```

Comment:
I think we should go with inclusive language here and call it `IsBlocklisted` or something along those lines.

Reply:
done

Reply:
thx